### PR TITLE
Feat 3 badges

### PR DESCRIPTION
**Pull request by SinchanaMS**
Created the badges component.

- Badges on Icons
- Badges on Avatars

image

### CODE REVIEW

File: stylesheets/components/badges/badges.css [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    padding: 1rem;
+}
+
+.avatar-badge{
+    display: flex;
+    position: relative;
+}
+
+.av-badge{
+    right: 1px;
```
Comment (aiman-ansari) [inline stylesheets/components/badges/badges.css]: Give them different name because both looks same.

**Review by aiman-ansari — APPROVED** [review]
Everything looks good . Do changes which I commented then you can merge. it.

File: stylesheets/components/badges/badges.css [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+}
+
+.grey-badge{
+    background: var(--grey-badge);
+}
+
+.green-badge {
+    background: var(--green-badge);
+}
+
```
Comment (Ramandeep9898) [inline stylesheets/components/badges/badges.css]: just a advise.... name the classes that are just changing color lil flexible...
bcoz u will have many component that will be of same color... (like green alert, green badge)
although it's just a advise u can always go as u please :

File: docs/badges.html [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+                        </span>
+                        <div class="green-badge badge icon-badge"></div>
+                    </div>
+
+                </div>
+
+            </div>
+
+
+            <div class="component-wrapper badges-on-avatars">
```
Comment (manojsarna) [inline docs/badges.html]: improve spacing!!

File: stylesheets/components/badges/badges.css [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+.icon-with-badge {
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+}
+
+.badge-icon-showcase {
+    position: relative;
+}
```
Comment (manojsarna) [inline stylesheets/components/badges/badges.css]: improve badge sizes with 2 digit numbers

File: docs/badges.html [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+        <header class="doc-header">
+            <h1 class="header-title">Clay</h1>
+        </header>
+        <nav class="nav-bar">
+            <h1 class="nav-title">Components</h1>
+            <a href="#" class="nav-subtitles">Avatars</a>
+            <a href="#" class="nav-subtitles">Alerts</a>
+            <a href="#" class="nav-subtitles">Badges</a>
+            <a href="#" class="nav-subtitles">Buttons</a>
+            <a href="#" class="nav-subtitles">Cards</a>
```
Comment (manojsarna) [inline docs/badges.html]: positioning is great!!!

File: docs/badges.html [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+                <h1 class="wrapper-heading badges-heading">Badges</h1>
+                <hr>
+                <div class="description wrapper-description badges-description">
+                    Badges are small circles, positioned either at the top-right or bottom-right of the parent
+                    component.
+                    They
+                    can be used to display notifications, updates, and users' status depending on where they are used.
+                </div>
+            </div>
+
```
Comment (manojsarna) [inline docs/badges.html:45]: dont use margins and padding on the components directly.

File: stylesheets/components/badges/badges.css [hunk 7]
```diff
@@ -0,0 +1,83 @@
+@import "/stylesheets/index.css";
+@import "/stylesheets/components/avatars/avatars.css";
+
+
+.general-badges {
+    margin-bottom: 0;
+}
```
Comment (manojsarna) [inline stylesheets/components/badges/badges.css]: Suggestion :you can reset elemrents

**Review by manojsarna — APPROVED** [review]
Overall Good Design and Code!!! LGTM!!

GTM!!

**Review by m26unkwn — APPROVED** [review]
Looks great to me good to merge.

File: stylesheets/components/badges/badges.css (hunk 1 above)
Comment (j24m) [reply stylesheets/components/badges/badges.css]: +1

**Comment by j24m** [comment]
Hey, UI looks so beautiful 😍, and the code is readable except for the naming conventions of classes in some components, rest is GTM!! Good work really.

File: stylesheets/components/badges/badges.css (hunk 1 above)
Comment (SinchanaMS) [reply stylesheets/components/badges/badges.css]: Thank you for pointing this out. Done.

File: stylesheets/components/badges/badges.css (hunk 2 above)
Comment (SinchanaMS) [reply stylesheets/components/badges/badges.css]: Thanks, Ramandeep. But I have used different shades, hence different colors variables and classes. 
Will keep that in mind for the same ones!

**Comment by SinchanaMS** [comment]
Made the PR changes. Merging the branch to dev.

**Pull request merged**